### ISSUE TITLE
regression loss is 0.0000000

### ISSUE THREAD

**Issue by airqj** [xuannianz/EfficientDet#113]
when i train efficientdet for my data, the classification_loss is descrease, but regression_loss is 0.000．
train command is 
- python3 train.py --snapshot imagenet --phi 1 --gpu 0 --random-transform --compute-val-loss --freeze-backbone --batch-size 8 --steps 1000 csv ~/DATA/specLabel/efficientdet_train.csv classes_map.txt --val-annotations-path ~/DATA/specLabel/efficientdet_val.csv    

where is the probleam?
![Screenshot at 2020-03-25 20-07-20](https://user-images.githubusercontent.com/2778047/77535166-1bb29800-6ed5-11ea-80f4-08139ec0c2a2.png)

**Comment by airqj** [comment]
change anchor ratio and size according to [https://github.com/martinzlocha/anchor-optimization/](url)
 and it works
